### PR TITLE
Add the legacy version of Master PDF Editor

### PR DESCRIPTION
**Pull request by jointri**
This package is important because version 5.0 introduced a watermark for editing PDF files without paying for the full version. Let me know if you found mistakes with my attempt to package up this flatpak package.

### CODE REVIEW

File: net.codeindustry.MasterPDFEditor-Legacy.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Master PDF Editor 4
+Comment=Edit PDF files
+Exec=masterpdfeditor %f
+Terminal=false
+Icon=net.codeindustry.MasterPDFEditor
```
Comment (tinywrkb) [inline net.codeindustry.MasterPDFEditor-Legacy.desktop:6]:
```suggestion
Icon=net.codeindustry.MasterPDFEditor-Legacy
```

**Comment by hfiguiere** [comment]
runtime 5.13 is EOL.

File: net.codeindustry.MasterPDFEditor-Legacy.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.codeindustry.MasterPDFEditor-Legacy</id>
```
Comment (hfiguiere) [inline net.codeindustry.MasterPDFEditor-Legacy.metainfo.xml:3]: I'd rather go with a version number than `-Legacy`

**Comment by hfiguiere** [comment]
PR for new packages are to be made against `new-pr` branch.

**Comment by jointri** [comment]
Why did the PR close when I changed from master to pr-new?

**Comment by hfiguiere** [comment]
it is long lasting github bug.

**Comment by jointri** [comment]
What should I do now then? I can't make a new PR against my master branch again.

**Comment by hfiguiere** [comment]
but you can make a new PR against `new-pr`

**Pull request closed without merging**